Controller/Adminhtml/AbstractMenu.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Snowdog\Menu\Controller\Adminhtml;
+
+use Magento\Backend\App\Action;
+use Magento\Backend\Model\View\Result\Page;
+
+abstract class AbstractMenu extends Action
```

Comment:
What do you think of `MenuAction` instead of `AbstractMenu` ?

Reply:
Done